packages/workflow/src/records/handler/update.ts

```diff
@@ -52,6 +98,12 @@ export const updateRoute = createRoute({
     const token = getToken(request)
     const payload = validateRequest(requestSchema, request.payload)
 
+    const informantTypeOfBundle = getInformantType(record)
+    const payloadInformantType = payload.details.registration.informantType
+
+    if (informantTypeOfBundle !== payloadInformantType)
+      record = filterInformantSection(record)
```

Comment:
@Nil20 Please add a comment explaining why the informant section needs to be removed so that we remember the next time this code needs to be worked with